style.css

```diff
@@ -0,0 +1,330 @@
+margin-top: -30px;
+}
+.listcard {
+margin-left: 35px;
+width: 335px;
+height: 257px;
+}
+.card {
+border: solid #fdfafa;
+background-color: #FFFFFF;
+margin-top: 10px;
+padding-left: 5px;
+}
+.listtop {
+width: 335px;
+height: 4px;
+background-color: #25A59E;
+margin-left: -5px;
+}
+.title {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 28px;
+line-height: 50px;
+width: 295px;
+color: #262626;
+}
+.description {
+width: 295px;
+font-family: "urbanist";
+font-weight: 400;
+font-size: 15px;
+line-height: 23px;;
+}
+.text {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 15px;
+line-height: 23px;
+width: 84px;
+color: #25A59E;
+margin-left: 170px;
+}
+.forward {
+margin-left: 270px;
+transform: translateY(-20px);
+width: 16px;
+height: 16px;
+}
+.card-1 {
+  background-color: #25A59E;
+  margin-top: 15px;
+  padding-left: 5px;
+}
+.title-1 {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 28px;
+line-height: 40px;
+color: #FFFFFF;
+}
+.description-1 {
+font-family: "Urbanist";
+font-weight: 400;
+font-size: 16px;
+line-height: 24px;
+width: 295px;
+color: #FFFFFF;
+}
+.text-1 {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 15px;
+line-height: 17.66px;
+margin-left: 150px;
+color: #FFFFFF;
+}
+.memory {
+margin-top: 500px;
+margin-left: 40px;
+}
+.make {
+font-family: "gilda display";
+font-weight: 400;
+font-size: 40px;
+line-height:  52px;
+}
+.thousands {
+font-family: "urbanist";
+font-weight: 500;
+font-size: 16px;
+line-height: 24px;
+margin-top: -10px;
+}
+.arrow {
+width: 335px; 
+height: 350px;
+color: #25A59E;
+}
+.travellers {
+width: 335px;
+margin-left: -350px;
+transform: translateY(-100px);
+}
+.trip {
+margin-left: 40px;
+font-family: "gilda display";
+font-weight: 400;
+font-size: 28px;
+line-height: 52px;
+margin-top: -90px;
+color: #262626;
+width: 335px;
+}
+.center {
+margin-left: 140px;
+}
+.movie {
+width: 375px;
+margin-left: 40px;
+}
+.header-container {
+margin-left: 40px;
+}
+.offer {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 40px;
+line-height: 52px;
+width: 335px;
+}
+.click {
+width: 147px;
+font-family: "urbanist";
+font-weight: 600;
+font-size: 18px;
+line-height: 50px;
+color: #FFFFFF;
+background-color:  #25A59E;
+margin-top: -450px;
+top: 0;
+left: 0;
+border: 0px;
+}
+.notification {
+margin-left: 40px;
+width: 375px;
+margin-top: 80px;
+}
+.movement {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 16px;
+line-height: 18.83px;
+}
+.agent {
+font-family: "urbanist";
+font-weight: 500;
+font-size: 16px;
+line-height: 24px;
+color: #595959;
+}
+.social-link {
+margin-left: 50px;
+}
+.bird {
+margin-top: -40px;
+margin-left: 50px;
+}
+.youtube {
+margin-top: -40px;
+margin-left: 110px;
+}
+.header-links {
+margin-left: 40px;
+font-family: "urbanist";
+font-weight: 600;
+font-size: 16px;
+line-height: 24px;
+color: #595959;
+padding-top: 30px;
+}
+ul {
+list-style: none;
+display: flex;
+gap: 30px;
+margin-left: -30px;
+}
```

Comment:
Hi @musyelevateher,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    <img src="https://media1.giphy.com/media/gLu90OMjz4j3hAkJk2/giphy.gif?cid=47028fa8sx12d824xrvy8zo8bq3w00f23hoj11gqqzus7q0f&ep=v1_gifs&rid=giphy.gif&ct=g" alt="Remote Collaboration" style="width:100%;">
</div>

### Highlights
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._